tests/unit/charms/lte_core_interface/v0/test_lte_core_interface_requirer.py

```diff
@@ -23,9 +23,34 @@ def setUp(self):
         self.harness = testing.Harness(DummyLTECoreRequirerCharm)
         self.addCleanup(self.harness.cleanup)
         self.harness.begin()
+        self.maxDiff = None
 
     @patch(f"{BASE_CHARM_PATH}._on_lte_core_available")
-    def test_given_lte_core_information_not_in_relation_data_when_relation_changed_then_on_lte_core_available_not_called(  # noqa: E501
+    def test_given_lte_core_information_in_relation_data_is_empty_when_relation_changed_then_on_lte_core_available_not_called(  # noqa: E501
+        self, patch_on_lte_core_available
+    ):
+        mme_ipv4_address = ""
+        relation_id = self.harness.add_relation(self.relation_name, remote_app=self.remote_app)
+        remote_app_relation_data = {"mme_ipv4_address": mme_ipv4_address}
+        expected_log = "Provider relation data did not pass JSON schema validation."  # noqa: E501
+
+        self.harness.update_relation_data(
+            relation_id=relation_id,
+            app_or_unit=self.remote_app,
+            key_values=remote_app_relation_data,
+        )
+        with self.assertLogs() as captured:
```

Comment:
Why are we updating the relation data twice with the same content.

Reply:
Leftover, removed.